persistent-redis/persistent-redis.cabal

```diff
@@ -1,5 +1,5 @@
 name:            persistent-redis
-version:         2.5.2.2
+version:         2.5.2.3
```

Comment:
should possibly be a major bump, but only type-changing thing is in the internal module